edge-fqm/src/main/resources/karate-config.js

```diff
@@ -92,12 +92,12 @@ function fn() {
     // Config for FOLIO CI "folio-integration" public ec2- dns name
     config.baseUrl = 'http://' + env + ':9130';
     config.edgeUrl = 'http://' + env + ':8000';
-    config.apikey = 'eyJzIjoiZlU4ZDNkc0pKTCIsInQiOiJkaWt1IiwidSI6ImRpa3VfYWRtaW4ifQ==';
+    config.apikey = 'eyJzIjoibTE2M0k2NTRHZ1pWOVBMdnRTa1MiLCJ0IjoiZGlrdSIsInUiOiJkaWt1X2FkbWluIn0K';
```

Comment:
I'm skeptical that this will actually do anything (IIRC, with the ephemeral secure store, the salt in the API key is ignored, and the salt looks like the only that changed between the old and new keys), but it's worth a try ¯\\\_(ツ)_/¯